From aae7b1d7215fc2cc5c5ecc9f1ace637cff56425f Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 23 Sep 2011 20:36:23 +0800
Subject: [PATCH 358/678] ENGR00157538 remove VDIG_PLL setting in plug-in code

According to the datasheet, VDIG_PLL needs to be increased
to 1.3v for TO2.0. This operation has been done in the
low_level_init.S. Remove the duplicated code here.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 board/freescale/mx53_ard/flash_header.S  |    5 -----
 board/freescale/mx53_evk/flash_header.S  |    5 -----
 board/freescale/mx53_loco/flash_header.S |    5 -----
 board/freescale/mx53_smd/flash_header.S  |    5 -----
 4 files changed, 0 insertions(+), 20 deletions(-)

diff --git a/board/freescale/mx53_ard/flash_header.S b/board/freescale/mx53_ard/flash_header.S
index a85870c..ccb1019 100644
--- a/board/freescale/mx53_ard/flash_header.S
+++ b/board/freescale/mx53_ard/flash_header.S
@@ -208,11 +208,6 @@ plugin_start:
 	REG_LD_AND_STR_END(ESDCTL_BASE_ADDR)
 #endif
 
-	/* For TO2 only, increase LD0 for VDIG_PLL to 1.3V */
-	REG_LD_AND_STR_INIT(IOMUXC_BASE_ADDR)
-	REG_LD_AND_STR_OP(52, 0x4, 0x00194005)
-	REG_LD_AND_STR_END(IOMUXC_BASE_ADDR)
-
 /*
  *  The following is to fill in those arguments for this ROM function
  * pu_irom_hwcnfg_setup(void **start, size_t *bytes, const void *boot_data)
diff --git a/board/freescale/mx53_evk/flash_header.S b/board/freescale/mx53_evk/flash_header.S
index 5a8c2e2..6cadf3f 100644
--- a/board/freescale/mx53_evk/flash_header.S
+++ b/board/freescale/mx53_evk/flash_header.S
@@ -147,11 +147,6 @@ plugin_start:
 	REG_LD_AND_STR_OP(64, 0x040, 0x04b80003)
 	REG_LD_AND_STR_END(ESDCTL_BASE_ADDR)
 
-	/* For TO2 only, increase LD0 for VDIG_PLL to 1.3V */
-	REG_LD_AND_STR_INIT(IOMUXC_BASE_ADDR)
-	REG_LD_AND_STR_OP(52, 0x4, 0x00194005)
-	REG_LD_AND_STR_END(IOMUXC_BASE_ADDR)
-
 /*
  *  The following is to fill in those arguments for this ROM function
  * pu_irom_hwcnfg_setup(void **start, size_t *bytes, const void *boot_data)
diff --git a/board/freescale/mx53_loco/flash_header.S b/board/freescale/mx53_loco/flash_header.S
index 16adb99..5a9dc0f 100644
--- a/board/freescale/mx53_loco/flash_header.S
+++ b/board/freescale/mx53_loco/flash_header.S
@@ -134,11 +134,6 @@ plugin_start:
 	REG_LD_AND_STR_OP(51, 0x01C, 0x00000000)
 	REG_LD_AND_STR_END(ESDCTL_BASE_ADDR)
 
-	/* For TO2 only, increase LD0 for VDIG_PLL to 1.3V */
-	REG_LD_AND_STR_INIT(IOMUXC_BASE_ADDR)
-	REG_LD_AND_STR_OP(52, 0x4, 0x00194005)
-	REG_LD_AND_STR_END(IOMUXC_BASE_ADDR)
-
 /*
  *  The following is to fill in those arguments for this ROM function
  * pu_irom_hwcnfg_setup(void **start, size_t *bytes, const void *boot_data)
diff --git a/board/freescale/mx53_smd/flash_header.S b/board/freescale/mx53_smd/flash_header.S
index 1dc22a2..0411f1b 100644
--- a/board/freescale/mx53_smd/flash_header.S
+++ b/board/freescale/mx53_smd/flash_header.S
@@ -134,11 +134,6 @@ plugin_start:
 	REG_LD_AND_STR_OP(51, 0x01C, 0x00000000)
 	REG_LD_AND_STR_END(ESDCTL_BASE_ADDR)
 
-	/* For TO2 only, increase LD0 for VDIG_PLL to 1.3V */
-	REG_LD_AND_STR_INIT(IOMUXC_BASE_ADDR)
-	REG_LD_AND_STR_OP(52, 0x4, 0x00194005)
-	REG_LD_AND_STR_END(IOMUXC_BASE_ADDR)
-
 /*
  *  The following is to fill in those arguments for this ROM function
  * pu_irom_hwcnfg_setup(void **start, size_t *bytes, const void *boot_data)
-- 
1.7.7.4

